src/integTest/java/com/google/cloud/tools/gradle/appengine/AppEngineStandardPluginIntegrationTest.java

```diff
@@ -112,13 +118,6 @@ public void testDevAppServer_async() throws InterruptedException, IOException {
       Thread.sleep(8000);
```

Comment:
8 seconds seems like a long time to wait and adding that to the test's time could be wasteful. What about a utility function that periodically checks to see if the server is down yet, and if not, blocks until it is or a specified timeout has expired?